src/analysis/destruct.ml

```diff
@@ -44,6 +44,47 @@ let () =
     | _ -> None
   )
 
+module Path_utils : sig
+  val to_shortest_lid :
+    env:Env.t ->
+    ?name:string ->
+    env_check:(Longident.t -> Env.t -> 'a) -> Path.t -> Longident.t
+end = struct
+  let opens env =
+    let rec aux acc = function
+    | Env.Env_open (s, path) -> aux (path::acc) s
+    | s -> Option.(
+        map ~f:(aux acc) (Browse_misc.summary_prev s)
+        |> value ~default:acc
+      )
```

Comment:
I'm not OK with this non indentation

Reply:
Also, which `map` is that?

Reply:
It is Option's map of course 😅 
I will change that.

Reply:
Ah! I had missed that local open. Yes, please, sorry for the verbosity/repetition.